actions/create_uplink.py

```diff
@@ -15,7 +15,6 @@ def create_uplink(api_auth, parameters, contexts):
     :rtype: string
     """
     try:
-        site_type = parameters["SiteTypes"]
         city = parameters["City"]
```

Comment:
Should we also include the country just in case there are duplicate city names in different countries?
E.g. Sydney in Canada vs. Sydney in Australia